.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Question
+about: Ask, and we'll do our best to help you out!
+---
+
+Before raising a question, make sure to
+
+* Examine [the docs](https://cookiecutter-django.readthedocs.io/en/latest/)
+* Check out [Gitter](https://gitter.im/pydanny/cookiecutter-django)
+* Check out [StackOverflow](https://stackoverflow.com/questions/tagged/cookiecutter-django)
+
+
+## The question
```

Comment:
The old template said: "for support questions, please use the `cookiecutter-django` tag on stackoverflow", I think we should keep directing people there.

Reply:
> for support questions, please use the cookiecutter-django tag on stackoverflow

always sounded a bit ambigous to me: why would people still come and create tickets here even though other channels exist? To my mind, everyone would be better off expressing their concerns via a single conventional channel -- Issues, in that case. 
I might be missing on something here, of course...

Reply:
The purpose of the other channels is that they require less bandwidth from the maintainers, anyone can answer questions on Stackoverflow or Gitter without us having to triage a record on Github. 

Also, having a question about cookiecutter-django often actually means a question about Django itself or a library we depend upon. Stackoverflow is a better channel for that type of things, where other people can help out.

Reply:
Yes, you've got a point.